tests/cpp_api_tests/rocAL_audio_unittests/rocAL_audio_unittests.cpp

```diff
@@ -200,13 +225,25 @@ int test(int test_case, const char *path, int qa_mode, int downmix, int gpu) {
         for (uint idx = 0; idx < output_tensor_list->size(); idx++) {
             buffer = static_cast<float *>(output_tensor_list->at(idx)->buffer());
             output_tensor_list->at(idx)->copy_roi(roi.data());
-            frames = roi[idx * 4 + 2];
+            if (output_tensor_list->at(idx)->dims().at(2) == 1)
+            {
+                max_samples = 1;
```

Comment:
Also if the dims at(2) is one, max_samples will not be 1, it needs to be output_tensor_list->at(idx)->dims().at(1) Please check